packages/amplify_auth_cognito_plugin/amplify_auth_cognito_ios/example/ios/Podfile

```diff
@@ -38,6 +38,8 @@ target 'unit_tests' do
   use_frameworks!
   use_modular_headers!
 
+  pod 'amplify_core', :path => './../../../../amplify_core/ios'
```

Comment:
This shouldn't be needed and if it is, then we need to figure out why

Reply:
Previously I needed to add this, but I can't remember why. Seems to work locally, so let's see if it passes in circle.

Reply:
@dnys1 Made the change!